src/libraries/System.Security.Cryptography/ref/System.Security.Cryptography.cs

```diff
@@ -2881,14 +2881,16 @@ protected virtual void Dispose(bool disposing) { }
         public static System.Security.Cryptography.SlhDsa ImportSlhDsaSecretKey(System.Security.Cryptography.SlhDsaAlgorithm algorithm, System.ReadOnlySpan<byte> source) { throw null; }
         public static System.Security.Cryptography.SlhDsa ImportSubjectPublicKeyInfo(byte[] source) { throw null; }
         public static System.Security.Cryptography.SlhDsa ImportSubjectPublicKeyInfo(System.ReadOnlySpan<byte> source) { throw null; }
-        public int SignData(System.ReadOnlySpan<byte> data, System.Span<byte> destination, System.ReadOnlySpan<byte> context = default(System.ReadOnlySpan<byte>)) { throw null; }
+        public byte[] SignData(byte[] data, byte[]? context = null) { throw null; }
+        public void SignData(System.ReadOnlySpan<byte> data, System.Span<byte> destination, System.ReadOnlySpan<byte> context = default(System.ReadOnlySpan<byte>)) { }
```

Comment:
nit: this should `{ throw null; }` in case we end up generating this file

Reply:
> this should { throw null; }

ApiGen doesn't add the `throw null` for `void` returning methods.

https://github.com/dotnet/runtime/blob/3b9b4222e6d0d93b34d4eb9ca3c402b55708dbbd/src/libraries/System.Security.Cryptography/ref/System.Security.Cryptography.cs#L66

Reply:
Yep, this was what the tool generated.